Model/Product/Builder.php

```diff
@@ -358,9 +359,20 @@ private function amendAttributeTags(Product $product, NostoProduct $nostoProduct
                     if (empty($attributeValue)) {
                         continue;
                     }
-                    //addTag1(), addTag2() and addTag3() are called
-                    $addTagMethodName = 'add' . $tag;
-                    $nostoProduct->$addTagMethodName(sprintf('%s:%s', $productAttribute, $attributeValue));
+
+                    switch ($tag) {
+                        case 'tag1':
```

Comment:
We should add constants for these into the SDK. I suppose we would do similar changes than this to other plugins as well.

Reply:
Let's do this later